orchagent/Makefile.am

```diff
@@ -2,6 +2,12 @@ INCLUDES = -I $(top_srcdir)
 
 CFLAGS_SAI = -I /usr/include/sai
 
+swssdir = $(datadir)/swss
+
+dist_swss_DATA = \
+		 duration_criteria.lua  \
```

Comment:
better to rename to mellanox_pfc_storm_detect_criteria.lua since the criteria is mellanox specific.